### PR TITLE
[wip]Paper: reference formatting

### PR DESCRIPTION
**Pull request by beyond2void**
- List all references in its right order
- Make 'note' consistent in reference
- Add Strenda&Enzyme Schema reference

### CODE REVIEW

File: paper/literature.bib [hunk 1]
```diff
@@ -385,103 +396,148 @@ @INPROCEEDINGS{1173142
...
+@misc{pyEnzyme,
+    title = {Lightweight syntax for rapid development of data management solution in enzymology and biocatalysis.},
+    url = {https://github.com/EnzymeML/PyEnzyme},
+    journal = {Github},
+    author = {Jan Range},
+    note = {[Accessed 15-10-2023]}
+}
+
+@misc{strenda,
+    title = {Schema provided by Jan Range},
```
Comment (PaulBredl) [inline paper/literature.bib:451]: This citation is a bit weird @Logende , do we really need it?
If so, @beyond2void the Author must be formatted like this: "Range, Jan"

File: paper/literature.bib [hunk 2]
```diff
@@ -385,103 +396,148 @@ @INPROCEEDINGS{1173142
...
-    title = {{J}{S}{O}{N} {S}chema editor for {W}indows --- json-buddy.com},
-    howpublished = {\url{https://www.json-buddy.com/json-schema-editor.htm}},
+
+@misc{githubGitHubJsonsystemspublic,
+    author = {{J}ack Wootton},
+    title = {JSON Systems --- github.com},
+    howpublished = {\url{https://github.com/jsonsystems/public}},
     year = {},
-    note = {[Accessed 08-10-2023]},
+    note = {[Accessed 07-May-2023]},
```
Comment (PaulBredl) [inline paper/literature.bib:482]: Why did you put may here and not 05? (applies to the ones above and below too)

File: paper/literature.bib (hunk 1 above)
Comment (Logende) [reply paper/literature.bib:451]: I think it make sense to refer to the schema, but we should give the reference a more formal title

**Review by PaulBredl — APPROVED** [review]

**Comment by PaulBredl** [comment]
you can merge after you solved the merge conflicts @beyond2void

**Comment by PaulBredl** [comment]
@beyond2void I saw that you renamed some entries. Can you check if the pdf still works, i.e all references still work?

**Comment by beyond2void** [comment]
> @beyond2void I saw that you renamed some entries. Can you check if the pdf still works, i.e all references still work?

I think I'll create new pr to rework on them, there are indeed some changes may made references not working.

**Comment by PaulBredl** [comment]
> > @beyond2void I saw that you renamed some entries. Can you check if the pdf still works, i.e all references still work?
> 
> I think I'll create new pr to rework on them, there are indeed some changes may made references not working.

why not use this PR?

**Comment by beyond2void** [comment]
I would close it, and rework quickly on new pr.

**Pull request closed without merging**